RevenueCatUI/Templates/V2/Components/Tabs/TabsComponentView.swift

```diff
@@ -22,15 +22,16 @@ import SwiftUI
 class TabControlContext: ObservableObject {
 
     @Published
-    var selectedIndex: Int = 0
+    var selectedTabId: String = ""
 
     let controlStackViewModel: StackComponentViewModel
-    let tabControlStackViewModels: [StackComponentViewModel]
+    let tabIds: [String]
 
     init(controlStackViewModel: StackComponentViewModel,
-         tabControlStackViewModels: [StackComponentViewModel]) {
+         tabIds: [String]) {
         self.controlStackViewModel = controlStackViewModel
-        self.tabControlStackViewModels = tabControlStackViewModels
+        self.tabIds = tabIds
+        self._selectedTabId = .init(initialValue: tabIds.first ?? "")
```

Comment:
Removed `tabControlStackViewModels` as it wasn't being used anywhere.

`tabIds` are stored here for two reasons: to initialise `selectedTabId` with the first tab, and to be able to toggle between first and second in a future toggle component.